lib/xapi/problem.rb

```diff
@@ -63,7 +67,12 @@ def md
       path('%s.md' % slug)
     end
 
-    def path(f)
+    def path(file_name)
+      old_path = root_path(file_name)
+      File.exist?(old_path) ? old_path : File.join(meta_dir, file_name)
+    end
```

Comment:
Location ordering: It would be good to look first in the "new" path and then in the "old" path if it is not found.